### PR TITLE
Feat/styled lesson details component LESQ-142

### PR DESCRIPTION
**Pull request by k-huggs**
## Lesson Details Component

# Components

- Lesson Requirements and test file
- Lesson Helper refactor, test and story
- Lesson Details refactor

## Issue(s)

Fixes #LESQ-142

## How to test

1. Go to https://deploy-preview-1781--oak-web-application.netlify.thenational.academy

## Screenshots

How it used to look (delete if n/a):

Screenshot 2023-08-08 at 17 08 22

How it should now look:

Screenshot 2023-08-08 at 17 09 18

Screenshot 2023-08-08 at 17 00 15

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
Tiniest of changes. 
- Could the gap between the title and body text here be increased ever so slightly. The designs make it to be 8px I think? Just looks a bit cramped at the moment
Screenshot 2023-08-09 at 09 12 55
- Could the "Equipment" and "Content guidance" titles be changed to h3s? They aren't within key words so should maintain the same heading level as all the other sections imo
- Could the items under "content guidance" be part of a ul if there is more than one?

**Review by kimon-satan — CHANGES_REQUESTED** [review]
Lets have a brief chat about that styled box

File: src/components/LessonDetails/LessonDetails.tsx [hunk 1]
```diff
@@ -10,53 +11,81 @@ import CommonMisconceptions, {
...
   teacherTips: TeacherTip[] | null | undefined;
+  equipmentAndResources: Equipment[] | null | undefined;
+  contentGuidance: ContentGuidance[] | null | undefined;
+  supervisionLevel: string | null | undefined;
 };
 
+const StyledBox = styled(Box)`
+  margin-bottom: 48px;
+
+  // can't set margin-bottom to 0px on the last child because of the lint rule
```
[the remark this answers is not in the document]
Comment (k-huggs) [reply src/components/LessonDetails/LessonDetails.tsx]: We don't know which lesson will have which resources, so wanted to use the last-child pseudo class. 
Ignored the linting rul

**Review by jack-skerrett-oak — APPROVED** [review]

**Review by kimon-satan — APPROVED** [review]

**Pull request merged**